be/src/agent/agent_server.cpp

```diff
@@ -191,8 +216,6 @@ AgentServer::Impl::~Impl() {
     STOP_POOL(MOVE, _move_dir_workers);
     STOP_POOL(UPDATE_TABLET_META_INFO, _update_tablet_meta_info_workers);
 #undef STOP_POOL
-
-    _thread_pool_publish_version->wait();
```

Comment:
Why to remove the code line?

Reply:
`_thread_pool_publish_version->shutdown();` will wait all threads exit, don't need to wait here, already talked with @zhuming